docker-compose.yml

```diff
@@ -2,32 +2,35 @@ version: "3.8"
 
 services:
   web:
-    image: "${DHIS2_IMAGE:-dhis2/core-dev:local}"
+    image: "${DHIS2_IMAGE:-dhis2/core}"
+    restart: unless-stopped
     ports:
-      - 127.0.0.1:8080:8080 # DHIS2
-      - 127.0.0.1:8081:8081 # Debugger: connect using commandline flag -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8081
-      - 127.0.0.1:9010:9010 # JMX port (for example for VisualVM)
+      - 127.0.0.1:${DHIS2_PORT:-8080}:8080 
     volumes:
+      - files:/opt/dhis2
       - ./docker/dhis.conf:/opt/dhis2/dhis.conf:ro
       - ./docker/log4j2.xml:/opt/dhis2/log4j2.xml:ro
     environment:
-      JAVA_OPTS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8081 \
-              -Dlog4j2.configurationFile=/opt/dhis2/log4j2.xml
-              -Dcom.sun.management.jmxremote \
-              -Dcom.sun.management.jmxremote.port=9010 \
-              -Dcom.sun.management.jmxremote.local.only=false \
-              -Dcom.sun.management.jmxremote.authenticate=false \
-              -Dcom.sun.management.jmxremote.ssl=false"
+      JAVA_OPTS: "-Dlog4j2.configurationFile=/opt/dhis2/log4j2.xml"
     depends_on:
       db:
         condition: service_healthy
+    healthcheck:
+      test: ["CMD-SHELL", "wget -q -S -O /dev/null http://127.0.0.1:${DHIS2_PORT:-8080}/dhis-web-commons/security/login.action 2>&1| grep -q 'HTTP.*200'"]
+      start_period: 120s
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    security_opt:
+      - no-new-privileges:true
 
   db:
-    image: ghcr.io/baosystems/postgis:12-3.3
+    image: postgis/postgis
+    restart: unless-stopped
     ports:
       - 127.0.0.1:5432:5432
     volumes:
-      - db-dump:/docker-entrypoint-initdb.d/
+      - postgresql:/var/lib/postgresql
     environment:
       POSTGRES_USER: dhis
```

Comment:
I would recommend a stronger password here and the usage of a .env file.

The pattern we're using on the IM projects is to have a committed .env.example file in the root of the project which serves as an example. The readme and the file itself can contain some instructions if needed.